devicetypes/smartthings/aeon-multisensor-6.src/aeon-multisensor-6.groovy

```diff
@@ -142,29 +142,27 @@ def installed(){
 
 def updated() {
 	log.debug "Updated with settings: ${settings}"
-	log.debug "${device.displayName} is now ${device.latestValue("powerSource")}"
 
-
-	def powerSource = device.latestValue("powerSource")
-
-	if (!powerSource) { // Check to see if we have updated to new powerSource attr
```

Comment:
I know this was meant as migration code, but everyone's device has probably transitioned over by now.

Reply:
I'll defer to @mckeed  on this. It is probably fine now.